cdhweb/projects/templates/projects/project_detail.html

```diff
@@ -12,8 +12,8 @@ <h1>{{ project.title }}</h1>
 
 <div class="links">
   <ul>
-    {% for link in project.projectresource_set.all %}
-        {% with link_type=link.resource_type.name %}
+    {% for link in project.projectrelatedlink_set.all %}
```

Comment:
same deal here, you can use the related name `project.related_links`

Reply:
addressed by 9b415a9